src/main/java/stream/flarebot/flarebot/util/GeneralUtils.java

```diff
@@ -567,11 +568,11 @@ public static void handleMultiSelectionCommand(User sender, TextChannel channel,
 
     /**
      * Checks if paths exist in the given json
-     *
+     * <p>
      * Key of the {@link Pair} is a list of the paths that exist in the JSON
      * Value of the {@link Pair} is a list of the paths that don't exist in the JSON
      *
-     * @param json The JSON to check <b>Mustn't be null</b>
+     * @param json  The JSON to check <b>Mustn't be null</b>
      * @param paths The paths to check <b>Mustn't be null or empty</b>
      * @return
```

Comment:
Missing return value

Reply:
Not relevant to this PR, I'll tidy everything up in v4.2